test/integration/zika.test.js

```diff
@@ -1,20 +1,85 @@
+import { waitUntilScreenStable } from "./helpers";
+
 describe("Zika", () => {
-  it('displays the title on the page', async () => {
-    await goToZikaPage()
+  it("displays the title on the page", async () => {
+    await goToZikaPage();
     await expect(page).toMatch("Real-time tracking of Zika virus evolution");
   });
 
-  it('clicks `Play` on the `TransmissionsCard`', async () => {
-    await goToZikaPage()
+  it("clicks `Play` on the `TransmissionsCard`", async () => {
+    await goToZikaPage();
+
+    await expect(page).toClick("button", { text: "Play" });
+
+    await expect(page).toMatchElement("button", { text: "Pause" });
+  });
+
+  describe("Color by", () => {
+    describe("author", () => {
+      it("matches the screenshot", async () => {
+        await toMatchImageSnapshot("author", async () => {
+          await page.keyboard.press("ArrowUp");
+          await page.keyboard.press("Enter");
+        });
+      });
+    });
 
-    await expect(page).toMatchElement('#TransmissionsCard')
+    describe("country", () => {
+      it("matches the screenshot", async () => {
+        await toMatchImageSnapshot("country", () => {});
+      });
+    });
 
-    await expect(page).toClick('button', { text: 'Play' })
+    describe("date", () => {
+      it("matches the screenshot", async () => {
+        await toMatchImageSnapshot("date", async () => {
+          await page.keyboard.press("ArrowUp");
+          await page.keyboard.press("ArrowUp");
+          await page.keyboard.press("Enter");
+        });
+      });
+    });
 
-    await expect(page).toMatchElement('button', { text: 'Pause' })
-  })
+    describe("region", () => {
+      it("matches the screenshot", async () => {
+        await toMatchImageSnapshot("region", async () => {
```

Comment:
I like this approach, you're ensuring that specific components on the page match visually (this is quite nice).